src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,89 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as todoActions } from '../../features/currentTodo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo }) => {
+  const dispatch = useAppDispatch();
+  const testTodo = useAppSelector(state => state.currentTodo);
+
+  // eslint-disable-next-line no-console
+  console.log(testTodo);
+
+  const { id, title, completed } = todo;
+
+  const [user, setUser] = useState<User | null>(null);
+  const fetchedUser = async () => {
```

Comment:
The name `fetchedUser` means that it's a variable. But it's a function so you need to use a verb here. `fetchUser` would be okay.